### PR TITLE
Generate timing.py file

### PR DESCRIPTION
**Pull request by subash-suresh**
What does this Pull Request accomplish?
This PR adds mako files and helper functions to generate `timing.py`

The following updates has been done in the following PR
- Created a `timing.py.mako` for the `timing.py` generation.
- Created a method called `get_enums_for_import` in the helpers to get a combined list of enums for the set of functions and attributes used.
- Updated the enum merge set with the enum name updates
- Fixed the `property_deprecated_template.py.mako` from generating unwanted new lines.
    -  This fix removes unwanted new lines from `ai_channel.py` and `ci_channel.py`.
- The following properties were generated newly, 
    - `first_samp_clk_offset`
    - `first_samp_clk_timescale`
    - `first_samp_clk_when`
    - `first_samp_timestamp_val`
    - `sync_pulse_time_when`
    - `timing_sync_pulse_force`
But currently I have blacklisted these new properties as they were not present before.

Why should this Pull Request be merged?
Implemented  [Task 2316154](https://dev.azure.com/ni/DevCentral/_workitems/edit/2316154): Generate the timing.py file

### CODE REVIEW

**Review by keer1111 — APPROVED** [review]

File: generated/nidaqmx/_task_modules/timing.py [hunk 1]
```diff
@@ -2892,6 +2892,7 @@ def sync_pulse_type(self):
             self._handle)
         check_for_error(error_code)
 
+
     def cfg_burst_handshaking_timing_export_clock(
```
Comment (bkeryan) [inline generated/nidaqmx/_task_modules/timing.py:2896]: https://github.com/ni/nidaqmx-python/blob/master/src/codegen/templates/property_deprecated_template.py.mako adds an extra newline. I think the fix is to add a backslash to the last line: `</%def>\`

**Review by bkeryan — APPROVED** [review]
Approved with suggestions

File: generated/nidaqmx/_task_modules/timing.py (hunk 1 above)
Comment (subash-suresh) [reply generated/nidaqmx/_task_modules/timing.py]: I have fixed this issue by adding a `\` in a `def` statement in the `property_deprecated_template.py.mako`, which seemed to be causing the issue.

**Comment by subash-suresh** [comment]
> same deal - if you retarget to master, let me know, and I can merge.

I have retargeted the branch to master

File: generated/nidaqmx/_task_modules/timing.py (hunk 1 above)
Comment (subash-suresh) [reply generated/nidaqmx/_task_modules/timing.py]: This fix also removes unwanted lines that were generated in `ai_channel.py` and `ci_channel.py`.

**Comment by zhindes** [comment]
@subash-suresh - there is a merge conflict in `src/codegen/properties/attribute.py `

**Comment by zhindes** [comment]
> @subash-suresh - there is a merge conflict in `src/codegen/properties/attribute.py `

Oh and now ` src/codegen/metadata/script_info.py `

**Pull request merged**